auth-tokens/src/main/java/com/palantir/tokens/auth/BearerTokens.java

```diff
@@ -78,7 +78,7 @@ public static BearerToken fromPath(Path path) throws IOException {
         }
 
         LOGGER.warn("No token file found in any of the configured paths: {}.", paths);
-        return Optional.absent();
+        return Optional.empty();
```

Comment:
I'm *strongly* against making a `tokens2` change just for this one method. I'd estimate that it'll take 30 hours over the course of months for projects to update to tokens2 and in the meanwhile we'll be converting between the 2 AuthHeaders everywhere. I'd even estimate somewhere in the 10's of developer hours wasted on dealing with conversions.

Can we just take the small backcompat-break hit, change this from guava to java8, and REMOVE the `tokens2` package alltogether?